src/advanced_inv.cpp

```diff
@@ -2136,7 +2136,7 @@ bool advanced_inventory::query_charges( aim_location destarea, const advanced_in
     amount = input_amount;
 
     // Includes moving from/to inventory and around on the map.
-    if( it.made_of( LIQUID ) ) {
+    if( it.type->phase == LIQUID ) {
```

Comment:
Would be nice if you kept the encapsulation by wrapping this into a function.

Reply:
I'm not opposed to that.  Thoughts on overloading `made_of` to take an optional boolean 'at_room_temp' or 'from_itype' or something like that?